x-pack/plugins/osquery/public/application.tsx

```diff
@@ -5,39 +5,30 @@
  * 2.0.
  */
 
-import { EuiErrorBoundary } from '@elastic/eui';
-import { euiLightVars, euiDarkVars } from '@kbn/ui-theme';
-import React, { useMemo } from 'react';
+import { EuiErrorBoundary, useEuiTheme } from '@elastic/eui';
+import React from 'react';
 import ReactDOM from 'react-dom';
 import { Router } from '@kbn/shared-ux-router';
 import { I18nProvider } from '@kbn/i18n-react';
-import { ThemeProvider } from 'styled-components';
+import { ThemeProvider as EmotionThemeProvider } from '@emotion/react';
 import { QueryClientProvider } from '@tanstack/react-query';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 
 import type { Storage } from '@kbn/kibana-utils-plugin/public';
 import type { AppMountParameters, CoreStart } from '@kbn/core/public';
-import { useUiSetting$, KibanaThemeProvider } from './shared_imports';
```

Comment:
I believe `KibanaThemeProvider` is providing `@emotion` `ThemeProvider`
https://elastic.github.io/eui/#/utilities/provider